pppd/plugins/pppol2tp/pppol2tp.c

```diff
@@ -241,10 +241,10 @@ static void send_config_pppol2tp(int mtu,
 		sprintf(&reorderto[0], "%d ", pppol2tp_reorder_timeout);
 	tid[0] = '\0';
 	if (pppol2tp_tunnel_id > 0)
-		sprintf(&tid[0], "%u ", pppol2tp_tunnel_id);
+		sprintf(&tid[0], "%u ", (short) pppol2tp_tunnel_id);
```

Comment:
This change is for sure incorrect. C printf modifier `%u` expects `unsigned int` type, but passed type is `short signed int`. Compiler should throw a warning about it.

Reply:
You are right, updated the change list. Must have eyeballed it when copying it from the original patch set. Thanks for the review.